homeassistant/components/homeassistant/strings.json

```diff
@@ -1,6 +1,7 @@
 {
   "config": {
     "abort": {
+      "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
```

Comment:
This new string is needed due to `translation_domain=HOMEASSISTANT_DOMAIN` in the `async def async_init` duplicate check